tools/shaderc/shaderc_spirv.cpp

```diff
@@ -1050,98 +1165,93 @@ namespace bgfx { namespace spirv
 					for (auto &resource : resourcesrefl.separate_images)
 					{
 						std::string name = refl.get_name(resource.id);
-						if (name.size() > 7
-						&&  0 == bx::strCmp(name.c_str() + name.length() - 7, "Texture") )
+						
+						if (name.size() > 7 && 0 == bx::strCmp(name.c_str() + name.length() - 7, "Texture") )
```

Comment:
I intentionally fold on if so it's easier to read, instead having one long line...

Reply:
Whoops, fixed